features/support/env.rb

```diff
@@ -8,6 +9,8 @@
 ActionController::Base.allow_rescue = false
 
 begin
+  require 'database_cleaner/cucumber'
+  DatabaseCleaner.clean_with :truncation
```

Comment:
I don't mind this change, but I would like a clarification on why you made it, please.

Reply:
The database wasn't being cleaned properly without these changes, we kept getting this 
![screen shot 2018-03-18 at 9 05 06 am](https://user-images.githubusercontent.com/34248712/37563918-964dc070-2a8b-11e8-968a-37911ab86c5b.png)
But we'll remove one of the `require` lines

Reply:
Okay. I'm pretty convinced toy can remove them both.

Reply:
Then we get this
![screen shot 2018-03-18 at 9 36 08 am](https://user-images.githubusercontent.com/34248712/37564134-d586a9b0-2a8f-11e8-803d-90cf6c869739.png)

Reply:
It works with only the ` DatabaseCleaner.clean_with :truncation` line so I'll get rid of the rest, but we need the truncation one

Reply:
Yes, you do.